frontend/cypress/support/helpers.js

```diff
@@ -0,0 +1,9 @@
+// https://glebbahmutov.com/cypress-examples/6.5.0/recipes/form-input-by-label.html#reusable-function
+export const getInputByLabel = (label) => {
+  return cy
+    .contains('label', label)
+    .invoke('attr', 'for')
+    .then((id) => {
+      cy.get('#' + id)
+    })
+}
```

Comment:
consider using `cy.findByRole('textbox', ...)`

for testing-library, but confirms this is the right role and possible caveat: https://stackoverflow.com/questions/64864453/unable-to-get-the-name-of-the-input-when-using-getbyrole-while-testing-with-test

Reply:
I think the findByRole approach might be so short that it no longer warrants a helper or command?

And it's a little fuzzy, but I'd recommend using custom commands (in `frontend/cypress/support/commands.js`) for something like this that seems to be valuable to use elsewhere. Custom functions are introduced pretty early in cypress docs, and it seems the the expectation is they'll do a lot of lifting. (I'm a little unsure why the official docs recommended helper functions for the graphql stuff, but I'm guessing it's just due to `cy.intercept` being a bit unique, and maybe better to batch 🤔 )

Reply:
Yeah, the findByRole with name does the same thing as the helper so I removed it in https://github.com/GCTC-NTGC/gc-digital-talent/pull/3084/commits/383b3126d8c6c36e4ef791e4cb651144a270797b